### ISSUE TITLE
Improve Cython version error

### ISSUE THREAD

**Issue by niboshi** [cupy/cupy#1364]
I'm seeing this error even if I have latest Cython (0.28.3) installed.

```
    RuntimeError: Missing file: cupy/core/_dtype.cpp
    Please install Cython 0.26.1. Please also check the version of Cython.
    See https://docs-cupy.chainer.org/en/stable/install.html
```
I think Cython version is irrelevant in my case.
We could improve the message.

**Comment by kmaehashi** [comment]
Hmm, I couldn't reproduce this error.

**Comment by kmaehashi** [comment]
Fixed via #4619
